pkg/trace/pb/remote_rates.go

```diff
@@ -0,0 +1,19 @@
+package pb
+
+// TargetTPS contains the targeted traces per second the agent should try to sample for a particular service and env
+type TargetTPS struct {
+	Service string `msgpack:"0"`
+	Env     string `msgpack:"1"`
+	// Value contains the targetTPS value to apply (target traces per second).
+	Value float64 `msgpack:"2"`
+	// Rank is the rank associated to this TargetTPS. Lower ranks of a same (env, service) are discarded
+	// in favor of the highest rank.
+	Rank uint32 `msgpack:"3"`
+	// Mechanism is the identifier of the mechanism that generated this TargetTPS
```

Comment:
Sorry, but this is still not clear enough. What are the possible options? Can you give some examples please?

Reply:
I think it's better to leave it to Barbayar as he needs to rebase on this PR and is waiting for it

Reply:
Yes, I can modify it in my PR.